Carta.py

```diff
@@ -343,67 +357,153 @@ def opponentRespondsInGrabPhase(self):
             else:
                 print("Carta.py: opponentRespondsInGrabPhase: opponent does not touch any card")
 
+    def removeCorrectGrabbingCard(self):
+        for card in self.grabbingCardsInPlay:
+            if (card.getLastWord() == self.curReadingCard.getLastWord()):
+                card.setFrame(None)
+                card.setStatus(GrabCardStatus.INVALID)
+                self.usedGrabbingCards.cards.append(card)
+                self.grabbingCardsInPlay.remove(card)
```

Comment:
1) modify `self.occupied` and `numOccupiedFrames`
2) keep track of correct grab card

correct grab card status --> correct grab card